content/02.body.md

```diff
@@ -64,6 +64,27 @@ Further discussion of this approximate edge prior and an derivation are availabl
 
 ### Prediction tasks
 
+We performed three prediction tasks to assess the performance of the edge prior, and we compared two additional features, a scaled product of source and target degree (scaled to the range [0, 1]), and our approximation of the edge prior.
+For the first two prediction tasks, we used 20 biomedical networks from the Hetionet heterogeneous network [@doi:10.7554/eLife.26726] that had at minimum 2000 edges.
+In the first task, we computed the degree-based prediction features (edge prior, scaled degree product, and prior approximation), and predicted the original edges in the network.
+Negative samples in the first task were node pairs in which an edge was not present in the original network.
+To assess the methods' predictive performances, we computed the area under the receiver operating characteristic curve for all three features.
+In the second task, we sampled 70% of edges from each of the networks, computed features on the sampled network, then attempted to predict held-out edges.
+For this task, negative samples were node pairs in which an edge did not exist in either original or sampled network, while positive samples were only those node pairs without an edge in the sampled network but with an edge in the original network.
+
+The third task evaluated the ability of the edge prior to generalize to new degree distributions.
+We used two domains---protein-protein interactions (PPI) and transcription factor-target gene (TF-TG) relationships---where networks are available that were generated through different methods: literature curation of low-throughput, hypothesis-driven research and high-throughput, systematic, hypothesis-free experimentation.
```

Comment:
```suggestion
We used two domains---protein-protein interactions (PPI) and transcription factor-target gene (TF-TG) relationships---where networks were available that were generated through different methods: literature curation of low-throughput, hypothesis-driven research and high-throughput, systematic, hypothesis-free experimentation.
```
I really don't like this sentence because I find it unwieldy and hard to read. Have you considered breaking it up into two sentences?